SUPPORT.md

```diff
@@ -0,0 +1,21 @@
+# **Support**
```

Comment:
`**` is not necessary for title.

Reply:
Thanks for the comment, I will modify it.